ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/institutions/institution.component.ts

```diff
@@ -226,11 +246,13 @@ export class InstitutionComponent implements OnInit, OnChanges, OnDestroy {
         return this.providersServiceAgent.createMedicalProvider(this.studyGuid,
             this.normalizedInstitutionType, form).pipe(
                 tap(response => {
-                    const newAnswer = new ActivityInstitutionInfo(form.physicianName ? form.physicianName : '',
+                    const newAnswer = new ActivityInstitutionInfo(
+                        form.physicianName ? form.physicianName : '',
                         form.physicianName ? form.physicianName : '',
                         form.city ? form.city : '',
                         form.state ? form.state : '',
-                        response.medicalProviderGuid
+                        response.medicalProviderGuid,
+                        form.country ? form.country : ''
```

Comment:
**Optional:**
It can be simplified like -->` form.country || ''`

Reply:
yea right, I just went with their existing style